setup.py

```diff
@@ -221,6 +221,7 @@
     },
     install_requires=[
         'click==7.0',
+        'urllib3<2',
```

Comment:
we don't need to limit urllib3 now.

Reply:
Hi @liushilongbuaa  seems it's not fixed, I still see same issue in my local build, and it also failed the checker of PR https://github.com/sonic-net/sonic-swss/pull/2763.